aioftp/server.py

```diff
@@ -1643,7 +1644,7 @@ def get_paths(self, connection, path):
                 break
 
         nums = tuple(map(int, str.split(host, "."))) + (port >> 8, port & 0xff)
-        info.append(str.format("({})", str.join(",", map(str, nums))))
+        info += str.format(" ({})", str.join(",", map(str, nums)))
```

Comment:
I've just checked `lftp`, and it works fine with multi-line replies. ES have issue, you should write authors about this bug.